### PR TITLE
Extend PRR questionaire with two more questions

### PR DESCRIPTION
**Pull request by wojtek-t**

### CODE REVIEW

File: keps/NNNN-kep-template/README.md [hunk 1]
```diff
@@ -413,6 +417,10 @@ _This section must be completed when targeting beta graduation to a release._
   Longer term, we may want to require automated upgrade/rollback tests, but we
   are missing a bunch of machinery and tooling and do that now.
 
+* **Does the rollout is accompanied with any deprecations of featureds, APIs, fields
```
Comment (johnbelamaric) [inline keps/NNNN-kep-template/README.md:420]: Some typos, change to: "**Is the rollout accompanied by any deprecations of features, APIs, fields".

However, do you mean *deprecations* or *removals*? I think it's removals that are the real issue.

File: keps/NNNN-kep-template/README.md [hunk 2]
```diff
@@ -384,6 +384,10 @@ _This section must be completed when targeting alpha to a release._
     - Will enabling / disabling the feature require downtime or reprovisioning
       of a node? (Do not assume `Dynamic Kubelet Config` feature is enabled).
 
+* **Does enabling the feature changes any default behavior?**
```
[the remark this answers is not in the document]
Comment (wojtek-t) [reply keps/NNNN-kep-template/README.md]: done

File: keps/NNNN-kep-template/README.md (hunk 1 above)
Comment (wojtek-t) [reply keps/NNNN-kep-template/README.md]: Yeah - removals are what we care more about.

I changed to "deprecations and/or removals". Do you think it's good enough?

**Comment by wojtek-t** [comment]
@johnbelamaric - fixed typo; PTAL

**Pull request merged**